spec/requests/api/services_spec.rb

```diff
@@ -667,4 +667,37 @@ def expect_svc_with_vms
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  describe 'add_resource' do
+    let(:vm1) { FactoryGirl.create(:vm_vmware) }
+    let(:vm2) { FactoryGirl.create(:vm_vmware) }
+
+    it 'can add multiple vms by href with an appropriate role' do
+      api_basic_authorize(action_identifier(:services, :add_resource))
+      request = {
+        'action'    => 'add_resource',
+        'resources' => [
+          { 'href' => vms_url(vm1.id) },
+          { 'href' => vms_url(vm2.id) }
+        ]
+      }
+
+      run_post(services_url(svc.id), request)
+
+      expected = {
+        'success' => true,
+        'message' => "Assigned resources to Service id:#{svc.id} name:'#{svc.name}'"
+      }
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to eq(expected)
```

Comment:
Hi @jntullo can you add an expect to make sure the svc got the vms added to it ?  Thanks.